packages/x-data-grid-premium/src/hooks/features/rowGrouping/useGridRowGrouping.tsx

```diff
@@ -73,9 +74,6 @@ export const useGridRowGrouping = (
     changeEvent: 'rowGroupingModelChange',
   });
 
-  /**
-   * API METHODS
-   */
```

Comment:
I kinda like having these, I'd just replace the `**` if this change is for the jsdoc lint.

Reply:
Hmm, do you think we should settle on a specific format around this or just a one-liner comment instead of JSDoc style would be good to go? Something like:
```tsx
  // API METHODS
```

Reply:
I'd keep it as it is, the other files already follow the same structure with the multiline comment, just remove the jsdoc marker.